sdk/storage/storage-blob/swagger/README.md

```diff
@@ -1282,4 +1282,26 @@ directive:
       delete $.properties["AllowPermanentDelete"];
 ```
 
+### Add description for properties
+
+```yaml
```

Comment:
Better to add the documentation to the swagger file so every language can benefit from it.

Reply:
Of course. Should send a PR for swagger spec.